buildSrc/src/main/kotlin/com/bmuschko/gradle/docker/AdditionalArtifactsPlugin.kt

```diff
@@ -0,0 +1,34 @@
+package com.bmuschko.gradle.docker
+
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+import org.gradle.api.plugins.JavaPluginConvention
+import org.gradle.api.tasks.bundling.Jar
+import org.gradle.api.tasks.javadoc.Groovydoc
+import org.gradle.api.tasks.javadoc.Javadoc
+import org.gradle.kotlin.dsl.*
+
+class AdditionalArtifactsPlugin: Plugin<Project> {
+    override fun apply(project: Project): Unit = project.run {
+        val sourceSets = project.the<JavaPluginConvention>().sourceSets
```

Comment:
Can be simplified to `the<SourceSetContainer>()`

Reply:
done